content/posts/2017/2017-10-24-timing_in_R.Rmd

```diff
@@ -0,0 +1,152 @@
+---
+title: "Timing in R"
+author: "Theo Roe"
+date: 2017-10-24
+output: html_document
+draft: true
+slug: "timing_in_R"
+tags: [r, timing, microbenchmark, tictoc, System.time]
```

Comment:
Keep tags lower case. Use a hyphen to separate words.

Makes consistent naming easier.